distribution/lib/Standard/Snowflake/0.0.0-dev/src/Internal/Snowflake_Connection.enso

```diff
@@ -0,0 +1,298 @@
+from Standard.Base import all
```

Comment:
I guess only `Snowflake_Details` need to be visible from other modules. If so, I suggest to make this module `private`.